src/ui/pages/Menu/components/Settings/components/RecoverySeedPhrase/ConfirmModal.tsx

```diff
@@ -0,0 +1,159 @@
+import { IonCheckbox, IonItem, IonList } from "@ionic/react";
+import { useCallback, useMemo, useState } from "react";
+import { useSelector } from "react-redux";
+import { i18n } from "../../../../../../../i18n";
+import { getStateCache } from "../../../../../../../store/reducers/stateCache";
+import { OptionModal } from "../../../../../../components/OptionsModal";
+import { PageFooter } from "../../../../../../components/PageFooter";
+import { VerifyPasscode } from "../../../../../../components/VerifyPasscode";
+import {
+  ConditionItemProps,
+  ConfirmModalProps,
+} from "./RecoverySeedPhrase.types";
+import { VerifyPassword } from "../../../../../../components/VerifyPassword";
+import "./RecoverySeedPhrase.scss";
+
+const ConditionItem = ({
+  text,
+  index,
+  checked,
+  onClick,
+}: ConditionItemProps) => {
+  return (
+    <IonItem
+      data-testid={`condition-item-${index}`}
+      className="condition-item"
+      onClick={() => onClick(index)}
```

Comment:
Not sure why there is a spot in the item that is not clickable. See the video.

https://github.com/user-attachments/assets/c4a93aa9-1550-45d8-8b94-610d30bacda2

Reply:
I tried to log it and found it fired click event 2 times when you click on `ion-input` but only fire 1 time when you click on text or check box => state not change when event fire 2 times.

 It seem this is ionic bug on v7: https://github.com/ionic-team/ionic-framework/issues/27796
 
I tried to fix by set height of text and button to make it full height `ion-input`.